code/modules/projectiles/guns/projectile/toy.dm

```diff
@@ -0,0 +1,72 @@
+/obj/item/weapon/gun/projectile/automatic/toy
+	name = "foam force SMG"
+	desc = "A prototype three-round burst toy submachine gun. Ages 8 and up."
+	icon = 'icons/obj/guns/toy.dmi'
+	icon_state = "saber"
+	item_state = "gun"
+	mag_type = /obj/item/ammo_box/magazine/toy/smg
+	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
+	force = 0
+	throwforce = 0
+	burst_size = 3
+	can_suppress = 0
+	clumsy_check = 0
+	needs_permit = 0
+
+/obj/item/weapon/gun/projectile/automatic/toy/pistol
+	name = "foam force pistol"
+	desc = "A small, easily concealable toy handgun. Ages 8 and up."
+	icon_state = "pistol"
+	w_class = 2
+	mag_type = /obj/item/ammo_box/magazine/toy/pistol
+	fire_sound = 'sound/weapons/Gunshot.ogg'
+	can_suppress = 0
+	burst_size = 1
+	fire_delay = 0
+	action_button_name = null
+
+/obj/item/weapon/gun/projectile/automatic/toy/pistol/update_icon()
+	..()
+	icon_state = "[initial(icon_state)][chambered ? "" : "-e"]"
+

```

Comment:
You didn't include the base type of `/obj/item/weapon/gun/projectile/automatic/toy/pistol/riot`